packages/eslint-plugin-kolibri/lib/utils.js

```diff
@@ -211,4 +211,33 @@ module.exports = {
       });
     });
   },
+
+  /**
+   * Report unused translation definitions.
+   */
+  reportUnusedTranslations(context, definitions, uses) {
+    const unused = definitions.filter(prop => !uses.includes(prop.name));
+
+    unused.forEach(prop => {
+      context.report({
+        node: prop.node,
+        message: `Unused translation defined for ${prop.node.name}`,
```

Comment:
Have it match the style of the other errors, like `Unused message found in $trs: "<name>"`